assets/app/view/entity_order.rb

```diff
@@ -16,15 +17,38 @@ def render
         style['padding-left'] = '1rem'
       end
 
+      children = []
+      if entity.corporation?
+        logo_props = {
+          attrs: {
+            src: entity.logo,
+          },
+          style: {
+            'max-height': '1.2rem',
+            padding: '0 .4rem 0 0',
+          },
+        }
+        logo_container_props = {
+          style: {
+            height: '100%',
+            display: 'inline-block',
+            'vertical-align': 'middle',
+          },
+        }
+        children << h(:span, logo_container_props, [h(:img, logo_props)])
+      end
+
       owner = " (#{entity.owner.name})" if !entity.player? && entity.owner
+      children << h(:span, "#{entity.name}#{owner}")
 
-      h(:div, { style: style }, "#{entity.name}#{owner}")
+      h(:div, { style: style }, children)
     end
 
     props = {
       style: {
         margin: '1rem 0 1rem 0',
-        'font-size': 'large',
+        'font-size': '1.rem',
```

Comment:
1 rem?

Reply:
Stray keystroke, thanks for the catch. Fixed.